paddle/phi/kernels/onednn/conv_handler.h

```diff
@@ -401,6 +401,14 @@ class ConvOneDNNHandlerT
       conv_attr.set_scales_mask(DNNL_ARG_SRC, 0);
 
       auto wei_scales = ConvertToDNNLScales("Scale_weights");
+      // By oneDNN API definition:
+      // - For per-tensor quantization: the mask should be 0
+      // - For per-dimension quantization: the mask should be 1 <<
+      // dimension_index Here, wei_scales.size() != 1 means per-channel
+      // quantization, the channel index in oneDNN is always 0, so we use mask =
+      // 1 << 0. If the conv is group, the weights shape will be [g, oc/g, ic,
+      // h, w], we need to do scaling along both group dim and oc dim, so the
+      // mask = (1 << 0) + (1 << 1).
       int mask = wei_scales.size() == 1
                      ? 0
                      : (groups > 1 ? ((1 << 0) + (1 << 1)) : 1 << 0);
```

Comment:
Since paddle is C++14 maybe you want to switch from shifting to binary integer literal:
https://en.cppreference.com/w/cpp/language/integer_literal
or just use values like 3 and 1.
```suggestion
      // By oneDNN API definition:
      // - For per-tensor quantization: the mask should be 0
      // - For per-dimension quantization: the mask should be 1 <<
      // dimension_index Here, wei_scales.size() != 1 means per-channel
      // quantization, the channel index in oneDNN is always 0, so we use mask =
      // 1 << 0. If the conv is group, the weights shape will be [g, oc/g, ic,
      // h, w], we need to do scaling along both group dim and oc dim, so the
      // mask = 0b0011.
      int mask = wei_scales.size() == 1
                     ? 0
                     : (groups > 1 ? 0b0011 : 0b0001);
```

Reply:
Thank you for the suggestion. Shifting is used in onednn examples, so I would prefer to also use it here in this PR. oneDNN is guaranteed to be buildable with c++11, I think this is why it doesn't use binary-literal. But as you said, Paddle support c++14, I think we can try more advanced features in the integration code in the future. :)